packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts

```diff
@@ -247,20 +243,19 @@ test(testCase, async (t: Test) => {
   test(`${testCase} - ${fInvoke} - ${cWithoutParams}`, async (t2: Test) => {
     try {
       const parameters = {
+        contractName: "",
         keychainId: keychainPlugin.getKeychainId(),
         invocationType: EthContractInvocationType.Call,
         methodName: "sayHello",
         params: [],
         gas: 1000000,
-        signingCredential: {
+        web3SigningCredential: {
           ethAccount: whalePubKey,
           secret: whalePrivKey,
           type: Web3SigningCredentialType.PrivateKeyHex,
         },
       };
-      await apiClient.invokeContractV1(
-        (parameters as any) as InvokeContractV1Request,
-      );
+      await apiClient.invokeContractV1(parameters);
```

Comment:
@aldousalvarez Sorry, my bad. In this particular case the casting was necessary because of the nature of the assertion itself (negative case where we are knowingly sending in invalid input and the compiler must be told explicitly to leave us alone).
Here's the suggestion of mine on how to refactor it so that there is no linter warning but the test is still passing (right now it is failing in the CI):

```suggestion
      const parameters = {
        keychainId: keychainPlugin.getKeychainId(),
        invocationType: EthContractInvocationType.Call,
        methodName: "sayHello",
        params: [],
        gas: 1000000,
        web3SigningCredential: {
          ethAccount: whalePubKey,
          secret: whalePrivKey,
          type: Web3SigningCredentialType.PrivateKeyHex,
        },
      };
      // Casting here is necessary because we are knowingly sending in an
      // incorrect request to verify that the validation catches it.
      await apiClient.invokeContractV1(
        (parameters as unknown) as InvokeContractV1Request,
      );
```